universal_robot/58790ba/58790ba.bug

```diff
@@ -71,4 +71,4 @@ fault-codes:
   - BDO:DEPENDENCY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Why, @ipa-hsd ?
Assuming the driver crashes and everything else runs, `UNRESPONSIVE` would be due to user commands not having an effect? But then, other commands (from scripts, etc.) would not have any effect either. Maybe `LOSS-OF-FUNCTIONALITY`? @ChrisTimperley

Reply:
By "user commands", do you mean with a keyboard or joystick? But that would be a rare scenario to control a robot isn't it? Typically, a user command for navigation or manipulation would be to start / stop an application. Rest of the actions would mostly depend pre-programmed commands or commands based on sensor input. So my understanding of `UNRESPONSIVE` was that when a control software sends a command to robot to perform an action, and it does not receive any acknowledgement (because it crashed or it is stuck..). I didn't realize  _"nothing happens when I push the button!""_ was in the literal sense.

Reply:
Same reasoning as above, both Unresponsive and Loss of Functionality apply here.